docs/guidance/example.md

```diff
@@ -158,9 +158,33 @@ In the document example below, an environmental impact assessment is provided. T
 
 ```
 
+#### Forecasts and metrics
+
+Publish structured data on planned and actual physical and financial progress using `Metric` objects in the `forecasts` (planned progress) and `metrics` (actual progress) arrays. Refer to the [implementation progress reports documentation](../../cost/#implementation-progress-reports) for further detail.
+
+In the example below, you can compare the planned `forecast/observation` for 75% for physical progress completed with the actual `metric/observation` for 75% of phyiscal progress completed (note the difference in `period`).
```

Comment:
Shouldn't this be:

```suggestion
In the example below, you can compare the planned `forecasts/observations` for 75% of physical progress completed with the actual `metrics/observations` for 75% of physical progress completed (note the difference in `period`).
```

Reply:
I referred to a single `observation` in each case because the example extracts a single planned `observation` object from the `forecast` metrics array for 75% of progress (`jsonpointer: /projects/0/forecasts/0/observations/1`), to compare with a single actual `observation` object from the `metrics` metrics array for the same. (I used single observation objects to make the difference between them clearer).
 
The instructions in implementation progress reports documentation are also consistent with referring to a single observation object for each progress plan/update: _"For each progress update, add an Observation object to the Metric object’s .observations array"._

Does that make sense?

Reply:
We should not refer to fields that do not exist. If you want to use the singular, you will need to rewrite the sentence so that the singular word appears in the text, and the correct plural field is referred separately.

Reply:
Ah, I see, thanks!